packages/web-app-files/src/composables/fileListHeaderPosition/useFileListHeaderPosition.ts

```diff
@@ -18,7 +16,6 @@ export const useFileListHeaderPosition = (): { y: Ref; refresh: () => void } =>
 
   window.onresize = refresh
   onMounted(refresh)
-  watch(() => store.getters['Files/inProgress'].length > 0, refresh)
```

Comment:
I *think* this can be removed as the upload-progress info isn't showing up in the file list header anymore. Therefore no height changes on upload.

Reply:
yes, can be removed :heart: